### DISCUSSION TITLE
SNZB-02 - how to debug constant disconnections of this device

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#20458] [Devices]
hi, 
i am running multiple (more than 10) zigbee2mqqt coordinators which snzb-02 sensor. Host are rasbperry pi with different zigbee2mqtt versions / branches - regulary updated since more than a Year, we can assume they are up to date using dev branch mostly. 

zigbeee coordinator hardware is different: Sonoff dongle x1, Sonoff E-Dongle more than 5, conbee x2, and sth else don't remember exactly. 

Problem  - excact problem on all of my coordinator/hosts. 

https://www.zigbee2mqtt.io/devices/SNZB-02.html#sonoff-snzb-02 keeps dying randomly no matter of distance, lqi or other parameters. Other devices like aquara sensors (which is my replacment for this buggy snzb-02), routers, bulbs and so on are working stable and OK. What is happning:

1/ Devices is not leaving the network (still visible) but not reporting
2/ Reseting the device gives instant report and online status, but sometimes is giving "device left the network", then reseting once again joins the network. 
3/ Sometimes the device is not waking up - i need to force remove the device from zigbee2mqtt and ad it again - then it works ok. 

This situation is very common - more than 30% of my devices are constantly offline no matter how many repeaters/lqi/ distance. Locations are far away and it's a big pain to reset the devices. Itead support is missing the point, maybe there is a way to create a debuginng sandbox in one of my locations to check if its a bug in zigbee2mqtt or the device itself?
Any help will be appriciated

**Comment by Ra72xx** [comment]
Maybe the same as https://github.com/Koenkk/zigbee2mqtt/discussions/20323 ?

**Reply by JacekKac** [reply]
thank you - i am pretty sure this is not the issue - my problems exist since months, and zigbee2mqtt version doesn make any difference. All non SNZB-02 devices are working as expected on all host ( adapters / versions / firmwars) i strongy belive that this is a device specific issue hardware or zigbee2mqtt issue which this devices.
